.github/workflows/gito-code-review.yml

```
@@ -19,13 +19,18 @@ jobs:
       uses: actions/setup-python@v5
       with: { python-version: "3.13" }
 
+    - id: fetch-latest-gito
+      run: echo "hash=$(pip index versions gito | sha256sum | cut -d' ' -f1)" >> $GITHUB_OUTPUT
+
+    - uses: actions/cache@v4
+      with:
+        path: ~/.cache/pip
+        key: pip-gito-${{ steps.gito-version.outputs.hash }}
```

actionlint (1.7.7)

property "gito-version" is not defined in object type {fetch-latest-gito: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)